optimade/server/entry_collections.py

```diff
@@ -71,14 +71,14 @@ def __init__(
         resource_cls: EntryResource,
     ):
         super().__init__(collection, resource_cls)
-        self.transformer = MongoTransformer()
+        self.transformer = NewMongoTransformer()
 
         self.provider = CONFIG.provider
         self.provider_fields = CONFIG.provider_fields
         self.page_limit = CONFIG.page_limit
         self.parser = LarkParser(
-            version=(0, 9, 7)
-        )  # The MongoTransformer only supports v0.9.7 as the latest grammar
+            version=(0, 10, 0)
```

Comment:
Should `variant` be passed here as well?
```suggestion
            version=(0, 10, 0), variant="default"
```